CHANGELOG

```diff
@@ -53,12 +53,13 @@ Yosys 0.9 .. Yosys 0.9-dev
     - Added support for flip-flops with synchronous reset to synth_xilinx
     - Added support for flip-flops with reset and enable to synth_xilinx
     - Added "check -mapped"
-    - Added checking of SystemVerilog always block types (always_comb,
+    - Added checking of SystemVerilog always block types (always_comb, 
```

Comment:
Do we really want to force a line break in the rendered output here?

Reply:
Sorry. I didn't make the line break, but I added the space accidentally (because I also thought that the line jump is not needed and checked the line width). How do you prefer that I fix the problem? Removing the line jump in a new commit is ok?

Reply:
I removed the line jump (there are longest lines than the resulting).

Reply:
There was a conflict due to changes in master. I solved it reverting the change in that line.